docs/en/structured-data/catalog-service.md

```diff
@@ -51,12 +51,16 @@ supported under storage systems locations can be served and read through Alluxio
 
 ## Using {{feature_name}}
 
-### Alluxio configuration
+Here are the basic configuration parameters and ways to interact with {{feature_name}}. More details
```

Comment:
should we get rid of `{{feature_name}}` and actually replace it with something?

Reply:
removed